### PR TITLE
Remove Unused CL Value in Compressibility Drag Calculation

### PR DESCRIPTION
**Pull request by timdmacdo**
This removes an unused variable that can be confusing when scanning the code. It resolves the second part of #128 (the first part has been moved to #309).

### CODE REVIEW

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py [hunk 1]
```diff
@@ -53,11 +53,9 @@ def compressibility_drag_wing(state,settings,geometry):
     
     wing = geometry
     if wing.tag == 'main_wing':
-        wing_lifts = conditions.aerodynamics.lift_breakdown.compressible_wings # currently the total aircraft lift
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py]: The lift of each wing is given individually in SUAVE. This is a poor assumption to assume the main wing is the only source of lift.

You can see the lift computation here:
https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py#L174

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py [hunk 2]
```diff
@@ -53,11 +53,9 @@ def compressibility_drag_wing(state,settings,geometry):
     
     wing = geometry
     if wing.tag == 'main_wing':
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py]: We should make this if isinstance(wing,main_wing):

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py [hunk 3]
```diff
@@ -53,11 +53,9 @@ def compressibility_drag_wing(state,settings,geometry):
     
     wing = geometry
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py]: Is the geometry that is passed a wing? Or is it the full vehicle?

Is this run for each wing?

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py [hunk 4]
```diff
@@ -68,12 +66,6 @@ def compressibility_drag_wing(state,settings,geometry):
...
     t_c_w   = wing.thickness_to_chord
     sweep_w = wing.sweeps.quarter_chord
-    
-    # Currently uses vortex lattice model on all wings
-    if wing.tag=='main_wing':
-        cl_w = wing_lifts
-    else:
-        cl_w = 0
         
     cos_sweep = np.cos(sweep_w)
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py]: There's a commented out piece of code below ...

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py (hunk 1 above)
Comment (timdmacdo) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py]: I agree with this but I think it's better to have a clear assumption than to hide it.

Comment (timdmacdo) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py]: Eliminating the assumption is still noted as an enhancement in #309.

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py (hunk 3 above)
Comment (timdmacdo) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py]: Looks like it's run for each wing. See in [Fidelity_Zero](https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Analyses/Aerodynamics/Fidelity_Zero.py#L107).

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py (hunk 4 above)
Comment (timdmacdo) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py]: I've restored it to produce what I believe was the desired functionality.

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py (hunk 2 above)
Comment (timdmacdo) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py]: Done

**Comment by WallyMaier** [comment]
What was the decision on this?

**Comment by timdmacdo** [comment]
Still being investigated.

**Comment by planes** [comment]
Let's wait until the VLM is all pulled in. Potentially useful for low level analyses with simple geometries in the future.

**Comment by WallyMaier** [comment]
whats the status here?
@timdmacdo @planes

**Comment by WallyMaier** [comment]
@timdmacdo @planes how this pr doing?  could we review?

**Comment by mclarke2** [comment]
I am closing this branch @WallyMaier. The issue has been resolved in #346

**Pull request closed without merging**